external/easylogging++/easylogging++.cc

```diff
@@ -2130,24 +2130,23 @@ static int priority(Level level) {
   return 7;
 }
 
-bool VRegistry::allowed(Level level, const char* category) {
+bool VRegistry::allowed(Level level, const std::string &category) {
   base::threading::ScopedLock scopedLock(lock());
-  const std::string scategory = category;
-  const std::map<std::string, int>::const_iterator it = m_cached_allowed_categories.find(scategory);
+  const std::map<std::string, int>::const_iterator it = m_cached_allowed_categories.find(category);
   if (it != m_cached_allowed_categories.end())
     return priority(level) <= it->second;
-  if (m_categories.empty() || category == nullptr) {
+  if (m_categories.empty()) {
```

Comment:
Shouldn't ` || category == nullptr` be replaced by ` || category.empty()`?

Reply:
No, empty would be "". This can't happen here anyway.